app/services/sign_in/token_serializer.rb

```diff
@@ -119,5 +119,9 @@ def anti_csrf_token
     def client_config
       @client_config ||= session_container.client_config
     end
+
+    def refresh_token
```

Comment:
you should add `path` to this method name so it doesn't get confused with being the refresh token